app/controllers/likes_controller.rb

```diff
@@ -26,7 +26,7 @@ def create
         format.json { render :json => @like.as_api_response(:backbone), :status => 201 }
       end
     else
-      render :nothing => true, :status => 422
+      render text: I18n.t("javascripts.failed_to_like"), status: 422
```

Comment:
`failed_to_like` is not used from javascript anymore. So I'm unsure if we should remove it from the javascript translations. We don't need to render it to the <head> on every page then, but we also need to retranslate it?